index.html

```diff
@@ -0,0 +1,122 @@
+    </div>
+    <div class="dates-container">
+      <h2 class="dates">2015.10.15(THU)~16(FRI)</h2>
+      <h4 class="place">@ Informatica Technical Institute, Santa Cruz, Bolivia</h4>
+    </div>
+  </section>
+  <section class="program">
+    <h3 class="main-program">Main Program</h3>
+    <hr class="underline">
+    <div class="card-container">
+      <div class="card">
+        <img src="images/conference-icon.png" alt="Conference Icon" class="img-cards">
+        <h5 class="card-title">Lecture</h5>
+        <p class="card-text">Listen to lectures from speakers from around the world and learn about the latest trends in the world.
+        </p>
+      </div>
+      <div class="card">
+        <img src="images/exhibition-icon.png" alt="Exhibition Icon" class="img-cards">
+        <h5 class="card-title">CC exhibition</h5>
+        <p class="card-text">Let's meet the creations of artists from various fields who share the open spirit of CC.
+        </p>
+      </div>
+      <div class="card">
+        <img src="images/forum-icon.png" alt="Forum Icon" class="img-cards">
+        <h5 class="card-title">Forum</h5>
+        <p class="card-text">We have time to share our thoughts and opinions with experts by topic.</p>
+      </div>
+      <div class="card">
+        <img src="images/workshop-icon.png" alt="Workshop Icon" class="img-cards">
+        <h5 class="card-title">Workshop</h5>
+        <p class="card-text">You can create your own creations using open source instead of just looking at them.</p>
+      </div>
+      <div class="card">
+        <img src="images/party-icon.png" alt="Party Icon" class="img-cards">
+        <h5 class="card-title">CC Ignite</h5>
+        <p class="card-text">Create opportunities to freely network with CC personnel from around the world.</p>
+      </div>
+    </div>
+    <div class="footer-card">
+      <a href="#">View full program</a>
+    </div>
+  </section>
+   <!--This section was build with JavaScript-->
+  <section class="exhib-sections">
+    <h3 class="title">Our Team</h3>
+    <hr class="big-title-bar" />
+    <ul class="exhib-section-list"></ul>
+  </section>
+  <section class="partners">
+    <h3 class="our-partners">Our Partners</h3>
+    <hr class="underline">
+    <div class="logo-container">
+      <a class="sponsor-link" href="https://www.snapchat.com/" target="_blank" rel="noopener noreferrer"><i class="sponsor-icon fab fa-snapchat"></i></a>
+      <a class="sponsor-link" href="https://www.slack.com/" target="_blank" rel="noopener noreferrer"><i class="sponsor-icon fab fa-slack"></i></a>
+      <a class="sponsor-link" href="https://www.github.com/" target="_blank" rel="noopener noreferrer"><i class="sponsor-icon fab fa-github"></i></a>
+      <a class="sponsor-link" href="https://www.mozilla.org/" target="_blank" rel="noopener noreferrer"><i class="sponsor-icon fab fa-firefox-browser"></i></a>
+    </div>
+  </section>
```

Comment:
**Suggested Changes**
- [ ] The `hr` underline is a little lower compared to the design, please move it a little closer to the `Partners` text.
- [ ] The `icons` don't show on the hosted page as shown in the below screenshot. Also, please find the right icons or use text and style it as close as possible.
Yours
![partner](https://user-images.githubusercontent.com/7220167/143607773-a49a8b11-5a0d-4ff9-8f12-2d56c5f8478f.png)

Expected Design
![partners](https://user-images.githubusercontent.com/7220167/143607841-4f17a24b-3b3d-4af9-b639-5a0c6a5da854.png)